packages/components/src/confirm-dialog/component.tsx

```diff
@@ -75,8 +77,8 @@ function ConfirmDialog(
 		[ handleEvent, onConfirm ]
 	);
 
-	const cancelLabel = __( 'Cancel' );
-	const confirmLabel = __( 'OK' );
+	const cancelLabel = cancelButtonText ?? __( 'Cancel' );
+	const confirmLabel = confirmButtonText ?? : __( 'OK' );
```

Comment:
```suggestion
	const confirmLabel = confirmButtonText ??  __( 'OK' );
```

Reply:
@chad1008 just pushed a tiny commit to fix a typo/syntax error :)

Reply:
Lol my bad! 🙈